src/TabNavigator/index.js

```diff
@@ -68,7 +67,7 @@ export default class TabNavigator extends Component {
         >
           {enabledTabs.map((tabName) => (
             <BottomNavigation.Action
-              showLoadingState={hasUpdatedLoadingStates}
+              showLoadingState={true}
```

Comment:
We should probably remove `showLoadingState` from `@protonapp/react-native-material-ui`, once we're ready to remove the flag.